server/server.go

```diff
@@ -45,9 +51,11 @@ func readFormData(r *http.Request) string {
 
 	// Read the command line arguments (additional parameters passed).
 	for index := 1; index <= len(r.Form); index++ {
+		keyName := fmt.Sprintf("%s%d", "key", index)
 		argName := fmt.Sprintf("%s%d", "arg", index)
+		key := r.FormValue(keyName)
 		arg := r.FormValue(argName)
-		fmt.Println(arg)
+		fmt.Printf("%s = %s\n", key, arg)
```

Comment:
seems like other than printing the args, this for loop has no value.

Reply:
Currently it is only printing the arguments, but later code for handling those arguments will be added in that loop.

Reply:
in that case, get rid of the loop and it as part of the future PR.

Reply:
in that case, we should get rid of the loop in this PR but add it back in the future PR.